tests/php/ORM/DBDateTest.php

```diff
@@ -238,6 +238,19 @@ public function testDayOfMonth()
         $this->assertEquals('10th - 20th Oct 2000', $range);
     }
 
+    public function testFormatReplacesOrdinals()
+    {
+        $this->assertEquals('20th', DBField::create_field('Date', '2000-10-20')->Format('d{o}'));
```

Comment:
If you make {o} include the number itself you don't need any new code, just use DayOfMonth(true).

Reply:
I’m trying that, and getting segfaults 😕. Every time I call a method inside `Format()` that _isn’t_ prefixed with `get` I get a segfault...

Reply:
Oh of course, recursion. `DayOfMonth` calls `Format()`. I can probably guard against that with a `strpos()`